code/modules/modular_computers/file_system/programs/command/card.dm

```diff
@@ -233,8 +243,11 @@
 						if(!jobdatum)
 							to_chat(usr, "<span class='warning'>No log exists for this job: [t1]</span>")
```

Comment:
I know that the other spans in here here don't use the span macros, but if you're adding new ones they should use the span macros. I'm not going to ask you to clean up the whole file and add span macros for everything unless you really want to, just that new things being added to the codebase use the correct format.

Reply:
No problem, I'll change my code to use the macros. Didn't actually know about them until you mentioned it.

Reply:
Okay, I went ahead and changed all the span code in the file to use macros instead. It was a simple find/replace.